### DISCUSSION TITLE
How to disable Cut/Copy Command triggered by Ctrl+X/Ctrl+C

### DISCUSSION THREAD

**Discussion by 7400263** [icsharpcode/AvalonEdit#288] [Q&A] [answered]
Your help is greatly appreciated.

**Comment by jimfoye** [comment, marked as answer]
This might be helpful:

http://stackoverflow.com/questions/27206919/avalonedit-copy-takes-forever-for-large-hightlighted-text-files

**Reply by 7400263** [reply]
Thanks a lot for your help, the thread is really helpful. BTW, the solution is:
`       DataObject.AddCopyingHandler(avEditor, onEditorCopying);
       ...
        static public void onEditorCopying(object sender, DataObjectCopyingEventArgs e)
        {
            e.CancelCommand();
        }`
